R/taxonomy.R

```diff
@@ -209,6 +209,15 @@ setMethod("checkTaxonomy", signature = c(x = "SummarizedExperiment"),
     }
 )
 
+# Function to set taxonomy ranks
+.setTaxonomyRanks <- function(rank = TAXONOMY_RANKS) {
+        assign("TAXONOMY_RANKS", rank, envir = .GlobalEnv)
+}
+
+# Function to get taxonomy ranks
+.getTaxonomyRanks <- function(x) {
+    return(TAXONOMY_RANKS)
+}
```

Comment:
Add @export tag
Function names without the dot. Dot means that the function is internal and is not exported.